docker-compose.yml

```diff
@@ -97,6 +97,23 @@ services:
     labels:
       be.vliz.container.project: "LWUA"
       be.vliz.container.group: "services"
-      be.vliz.container.owner: "laurian.van.maledghem@vliz.be"
+      be.vliz.container.owner: "laurian.van.maldeghem@vliz.be"
       be.vliz.container.description: "LifeWatch User-Analysis Platform"
-      be.vliz.container.vcs-url: "https://github.com/lifewatch/user-analysis-2023.git"
\ No newline at end of file
+      be.vliz.container.vcs-url: "https://github.com/lifewatch/user-analysis-2023.git"
+
+  sembench:
+    build:
+      context: ./docker/lwua-sembench/
+    image: lwua/lwua_sembench
+    container_name: lwua_sembench
+    volumes:
+      - ./data:/data
+      - ./config:/config
+    environment:
+      - INPUT_DATA_LOCATION=/data/input-data
+      - OUTPUT_DATA_LOCATION=/data/output-data
+      - SEMBENCH_DATA_LOCATION=/data/sembench-data
```

Comment:
unclear role?

also: no path needed for templates?

Reply:
`sembench-data` is the folder that contains all data required to run pysembench. At the moment the templates should go in there, but e.g. shacl files as well